openedx/features/job_board/forms.py

```diff
@@ -16,3 +16,13 @@ class Meta:
             'compensation': RadioSelect,
             'hours': RadioSelect,
         }
+        labels = {
+            'city': 'City:',
+            'country': 'Country:',
```

Comment:
As we have already provided `verbose_name` in model, How about removing city and country labels  from here as they are same in model. Aren't we doing same thing again in form that is already done in model?

Reply:
`verbose_name` was not used because of the specific requirement of having `:` at the end of only these fields. Other fields do not have `:` at the end. If we use `verbose_name` then everywhere some fields will have `:` some will not.

This specific requirement is for this form only, so adding labels in form only makes sense here. What do you think?

Reply:
Alright, How about adding a comment so that in future no dev remove this considering it redundant? 
Rest looks fine to me.

Reply:
done